decidim-results/db/migrate/20170612101951_migrate_results_category.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class MigrateResultsCategory < ActiveRecord::Migration[5.1]
+  def change
+    Decidim::Results::Result.find_each do |result|
```

Comment:
What do you mean by "all this"? Migrating proposals, results, meetinds and projects in a single migration? I am not sure if that's good because every engine has its own migrations and I need to respect the order.. right?